### PR TITLE
Update Set-CsTeamsComplianceRecordingPolicy.md

### PR DESCRIPTION
**Pull request by 2kiran**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — CHANGES_REQUESTED** [review]
Please add the parameter to the syntax section. Thanks.

**Review by get-itips — CHANGES_REQUESTED** [review]
agreed with @dariomws

**Comment by yogkumgit** [comment]
@2kiran please implement the suggestion. Thanks

**Comment by yogkumgit** [comment]
@2kiran Gentle reminder, please update the PR. Thanks

File: skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md [hunk 1]
```diff
@@ -194,6 +195,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableComplianceRecordingAudioNotificationForCalls
+Setting this attribute to true disables recording audio notifications for 1:1 calls that are under compliance recording
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md:199]:
```suggestion
Setting this attribute to true disables recording audio notifications for 1:1 calls that are under compliance recording.
```

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward @get-itips

**Comment by get-itips** [comment]
The branch has now conflicts that prevent merge @2kiran can you review ? Thanks

**Comment by yogkumgit** [comment]
@tiburd Please take an appropriate next step. Thanks

**Comment by tiburd** [comment]
@2kiran 
Thank you for submitting this PR. After you make the requested updates, we'll continue reviewing it.

If the update isn't made soon, we might close this PR.

**Comment by 2kiran** [comment]
Hi Tim,
I did make the changes. Didn't they come through?

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**